### PR TITLE
Cleanups, fixes and a bit of optimizations for site/components batch #2

### PR DESCRIPTION
**Pull request by frankmayer**
- com_contact

Note: This is a single commit bundling all types of changes, since PR #12261 which had detailed commits, was rejected as a whole

### CODE REVIEW

File: components/com_contact/views/category/view.html.php [hunk 1]
```diff
@@ -62,7 +62,7 @@ public function display($tpl = null)
 			$item->slug = $item->alias ? ($item->id . ':' . $item->alias) : $item->id;
 			$temp       = new Registry;
 			$temp->loadString($item->params);
-			$item->params = clone($this->params);
+			$item->params = clone$this->params;
```
Comment (andrepereiradasilva) [inline components/com_contact/views/category/view.html.php]: can you add a space here?

Comment (frankmayer) [inline components/com_contact/views/category/view.html.php]: oops... ;)

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 2372fcbc20f304460aa37f5aa28047ab1796284d<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12291">issues.joomla.org/joomla-cms/12291</a>.</sub>

**Comment by wilsonge** [comment]
RTC on review for 3.6.4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12291">issues.joomla.org/tracker/joomla-cms/12291</a>.</sub>

**Comment by brianteeman** [comment]
Milestone changed to 3.7 as it is now not planned to have a 3.6.4 release

**Pull request merged**